include/constants.h

```diff
@@ -26,6 +26,7 @@ const char* const HTTP_API_KEY = "http.api.key";
 const char* const HTTP_API_HEADER_KEY_NAME = "http.api.header.key.name";
 const char* const AUDIT_ENABLED = "audit.enabled";
 const char* const AUDIT_OUTPUT_PATH = "audit.output.path";
+const char* const EUD_DURATION = "eud.duration";
```

Comment:
I feel like these belong in their own federated learning constants file

Reply:
that should also not be available if not compiled with federated learning compile time flag set to ON

Reply:
I think it's simpler to have all the configuration options in one file.

I've pushed a change to put most of the options in an `#ifdef` with the federated learning flag.